tools/cross_runner/kernels/conv_nchw_fp16.cpp

```diff
@@ -26,7 +26,7 @@ implied warranties, other than those that are expressly stated in the License.
 
 #define BLOCK_H 1
 #define BLOCK_BATCH 1
-#define BLOCK_OC 8
+#define BLOCK_OC 16
```

Comment:
Why increase OC size?

Reply:
it improves performance for most cases as the channel numbers are higher.